### ISSUE TITLE
`window.location` should be used instead of `document.location`

### ISSUE THREAD

**Issue by awwx** [meteor/meteor#2380]
If a frame, image, or form is named "location" then `document.location` will refer to that element instead of being an alias for `window.location`.  (http://stackoverflow.com/a/12098898/1756909)

```
<form name="location"></form>

> document.location
<form name=​"location">​
​</form>​
> window.location
Location {replace: function, assign: function, ...
```

`document.location` is currently used in

```
facebook/facebook_client.js
livedata/sockjs-0.3.4.js
jquery-history/history.js
jquery-history/history.html4.js
reload-safetybelt/safetybelt.js
test-in-console/driver.js
```

https://github.com/meteor/meteor/wiki/Meteor-Style-Guide should also be updated to recommend using `window.location` instead of `document.location`.

**Comment by hwillson** [comment]
I've opened PR #8223 to help address the remaining items in this issue. Thanks!

**Comment by abernix** [comment]
This has been fixed with #8223.  Thanks for reporting this and thanks for your contribution, as usual, @hwillson!
